tests/Controller/Admin/BlogControllerTest.php

```diff
@@ -159,8 +163,12 @@ public function testAdminEditPost(): void
 
         $this->assertResponseRedirects('/en/admin/post/1/edit', Response::HTTP_FOUND);
 
+        /** @var PostRepository $postRepository */
+        $postRepository = static::getContainer()->get(PostRepository::class);
+
         /** @var \App\Entity\Post $post */
```

Comment:
smae here. The types on PostRepository allow inferring that variable

Reply:
After removal of this PHPDoc new entry is added to baseline:
```
		-
			message: "#^Cannot call method getTitle\\(\\) on App\\\\Entity\\\\Post\\|null\\.$#"
			count: 1
			path: tests/Controller/Admin/BlogControllerTest.php
```
So do we need more entries in baseline or PHPDoc here?